<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>300</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>400</width>
    <height>300</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label_3">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>30</height>
      </size>
     </property>
     <property name="text">
      <string>Select a host to use (must be reachable by the PyRQ)</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="frame">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QPushButton" name="pushButton_Cancel">
      <property name="geometry">
       <rect>
        <x>277</x>
        <y>200</y>
        <width>81</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>Cancel</string>
      </property>
     </widget>
     <widget class="QListWidget" name="listWidget_Hosts">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>60</y>
        <width>341</width>
        <height>121</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_Refresh">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>200</y>
        <width>98</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>Refresh</string>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>20</y>
        <width>111</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Existing Host:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_existingHost">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>13</y>
        <width>221</width>
        <height>27</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_Accept">
      <property name="geometry">
       <rect>
        <x>187</x>
        <y>200</y>
        <width>81</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>Ok</string>
      </property>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
